### PR TITLE
Add Localization support for Publisher Login

### PR DESCRIPTION
**Pull request by sandalu95**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/ProtectedApp.jsx [hunk 1]
```diff
@@ -1,12 +1,29 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/ProtectedApp.jsx]: old license year

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/App.jsx [hunk 2]
```diff
@@ -73,6 +98,21 @@ class Publisher extends React.Component {
...
+    /**
+     * Load locale file.
+     *
+     * @param {string} locale Locale name
+     */
+    loadLocale(locale = 'en') {
+        fetch(`${Utils.CONST.CONTEXT_PATH}/public/app/locales/${locale}.json`)
+            .then(resp => resp.json())
+            .then((data) => {
+                // eslint-disable-next-line global-require, import/no-dynamic-require
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/App.jsx]: please remove commented line

Comment (sandalu95) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/App.jsx]: Comment is required to skip the eslint issue for no-dynamic-require

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/ProtectedApp.jsx (hunk 1 above)
Comment (sandalu95) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/ProtectedApp.jsx]: Changes resolved.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/App.jsx (hunk 2 above)
Comment (isharac) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/App.jsx]: +1  Since it is a configuration comment.

**Pull request merged**